doc/install.rst

```diff
@@ -28,45 +28,36 @@ You may choose to install ``mne-bids``
 Installation via pip
 --------------------
 
-Besides ``numpy`` and ``scipy`` (which are included in the standard Anaconda
-installation), you will need to install the most recent version of ``MNE``
-using the ``pip`` tool:
+To install MNE-BIDS including all dependencies required to use all features,
+simply run:
 
 .. code-block:: bash
 
-   $ pip install -U mne
+   pip install --user -U mne-bids[full]
 
+This ``pip`` command will also work if you want to upgrade if a newer version
+of ``mne-bids`` is available.
 
-Then install ``mne-bids``\ :
+If you don't require advanced features like interactive visual data inspection,
+you may also install a basic version of MNE-BIDS via
 
 .. code-block:: bash
 
-   $ pip install -U mne-bids
-
-
-These ``pip`` commands also work if you want to upgrade if a newer version of
-``mne-bids`` is available. If you do not have administrator privileges on the
-computer, use the ``--user`` flag with ``pip``.
+   pip install --user -U mne-bids
 
 To check if everything worked fine, the following command should not give any
 error messages:
 
 .. code-block:: bash
 
-   $ python -c 'import mne_bids'
-
-For full functionality of ``mne-bids``, you will also need to ``pip install``
-the following packages:
-
-- ``nibabel``, for interacting with MRI data
-- ``pybv``, to convert EEG data to BrainVision if input format is not valid according to EEG BIDS specifications
+   python -c 'import mne_bids'
 
-If you want to use the latest development version of ``mne-bids``, use the
-following command:
+MNE-BIDS works best with the latest stable release of MNE-Python. To ensure
+MNE-Python is up-to-date, run:
 
 .. code-block:: bash
 
-   $ pip install https://api.github.com/repos/mne-tools/mne-bids/zipball/master
+   pip install -U mne
```

Comment:
why did you remove this?

Reply:
I think if anything, it should go to the developer installation instructions, no? Normal users shouldn't have to run `master` – if they do, we should probably create a new release :)